packages/members-api/index.js

```diff
@@ -239,6 +240,46 @@ module.exports = function MembersApi({
         }
     });
 
+    middleware.cancelSubscription.use(ensureStripe, body.json(), async function (req, res) {
+        const identity = req.body.identity;
+        const subscriptionId = req.params.id;
+
+        let email;
+        try {
+            if (!identity) {
```

Comment:
This should throw IMO and result in a 401 status code

Reply:
Agreed :+1: